cypress/integration/advancedOption.test.ts

```diff
@@ -12,67 +12,65 @@ context("Advanced option test", () => {
   })
 
   function advancedOptionTest(poolName: PoolName) {
-    const host = Cypress.env("DAPP_HOST") as string
```

Comment:
move each loop into its own test block, rm the visit and wait from each which is redundant with the beforeAll